src/alephbet.coffee

```diff
@@ -67,6 +67,34 @@ class AlephBet
       @storage().get("#{@options.name}:variant")
 
     pick_variant: ->
+      # we are checking that all variants of experiment has weights
+      variants_has_weight = utils.checkWeights(@variants).every (contains_weight) -> contains_weight
```

Comment:
can this logic move into `utils` where anyway most of the code that check weights exists?

Reply:
... also, it should probably say `variants_have_weights`, since variants are plural, and maybe clearer would be `all_variants_have_weights` because you are making sure *all* of them have weights.

It might also make sense to log a warning if only some have weights, because then it's more likely to be a mistake and the test won't behave as expected.